config/routes.rb

```diff
@@ -24,9 +24,14 @@
       patch :start
     end
 
+    resources :comments do
```

Comment:
Suggest to move `comments` to nested routes to `cources`. It should resolve issue with passing `commentable_type` and `commentable_id` params.
Please take a look: https://guides.rubyonrails.org/routing.html#routing-concerns